Packs/AzureSentinel/Scripts/MicrosoftSentinelSubmitNewComment/MicrosoftSentinelSubmitNewComment.yml

```
@@ -0,0 +1,30 @@
+args:
+  - name: new_comment
+    required: true
+    description: The comment text to be added to the incident comments.
+  - name: incident_id
+    required: false
+    description: The ID of the incident to add the comment to. This argument is relevant only when the script is called directly from the War Room.
+commonfields:
+  id: MicrosoftSentinelSubmitNewComment
+  version: -1
+comment: This script is used to add a new comment to incident.
```

Not sure this is the right terminology but please use this template
```suggestion
comment: Use this script to add a comment which will then be mirrored as a comment to a Sentinal event. This script should be run within an incident.
```